src/View/View.php

```diff
@@ -301,9 +302,10 @@ public function renderFile(string $viewFile, array $parameters = [], ?ViewContex
         // TODO: these two match now
         $requestedFile = $viewFile;
 
-        if ($this->theme !== null) {
+        if (!empty($this->theme)) {
```

Comment:
Doesn't seem to be related.

Reply:
Phpstan suggests that it will never be null, with empty we validate that it is not null or empty.